src/diff/index.js

```diff
@@ -238,6 +238,7 @@ function diffElementNodes(dom, newVNode, oldVNode, context, isSvg, excessDomChil
 			dom.multiple = newProps.multiple;
 		}
 
+		newVNode._children = newVNode.props.children;
```

Comment:
I'm probably missing something - isn't this duplicative of line 51?

Reply:
Yea, looking again, this is one of the weirder parts of this change. `diffChildren` expects the children of the VNode to already be on the `_children` property. It then loops through the `_children` property and flattens the result which is re-assigned to `_children`.

I did it this way because the children of a VNode are different for Components vs DOM VNodes. Components' children are the result of render while DOM VNodes are the JSX children (`props.children`). To keep the logic of determining what the children are in this function I needed a way to pass the resolved children into `diffChildren`. I did this by sticking the resolved children onto `_children`.

So this line puts the JSX children onto `_children`. Line 118 is the corresponding line for Components.